dist/typings/proteus.d.ts

```diff
@@ -266,7 +266,7 @@ export declare module session {
   }
 
   abstract class PreKeyStore {
-    public prekeys: Array<number>;
+    public prekeys: Array<keys.PreKey>;
```

Comment:
`prekeys` is actually just an array which holds prekey IDs (of type `number`) which will then later be removed (or received) from it's `PreKeyStore`. Sample implementation: https://github.com/wireapp/wire-webapp-cryptobox/blob/v6.0.0/src/main/ts/store/ReadOnlyStore.ts#L8

Reply:
Hi @hayeah, I really like when peope are thinking about different use cases for our libraries. :)

If it supports your vision, then you can create a new PR to be able to disable automatic PreKey refillment. I will also assist you once an initial PR is there.

But first we need to solve this issue here. Please do the adjustments that I have requested and then we can get this PR merged.

Reply:
ok. I'll dig into this and make some changes on my fork. For now, opened an #70